source/render/BankPanelDraw.cpp

```diff
@@ -0,0 +1,237 @@
+/* BankPanelDraw.cpp
+Copyright (c) 2014 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "../BankPanel.h"
+
+#include "../text/Format.h"
+#include "../GameData.h"
+#include "../Information.h"
+#include "../Interface.h"
+#include "../PlayerInfo.h"
+#include "../text/Table.h"
+
+#include <string>
+
+using namespace std;
+
+namespace {
+	// Dimensions of the table.
+	constexpr int MIN_X = -310;
+	constexpr int MAX_X = 190;
+	
+	// Column headings.
+	constexpr const char *HEADING[6] = {"Type", "Principal", "Interest", "Term", "Payment", ""};
+	// X coordinates of the columns of the table.
+	constexpr int COLUMN[5] = {-290, -180, -100, -30, 20};
+	
+	// Position of the first line of the table.
+	constexpr int FIRST_Y = 78;
+	
+	// Maximum number of rows of mortages, etc. to draw.
+	constexpr int MAX_ROWS = 8;
+}
+
+
+
+// Draw the bank information.
+void BankPanel::Draw()
```

Comment:
I'm still not happy that the implementation of the BankPanel class is now divided over multiple files (BankPanel.cpp and BankPanelDraw.cpp). C++ does allow this construction, but most projects use the convention that the implementation of a class sits in a .cpp file with the name of the class and only in that file.